olwebsite-app/src/components/Carousel/Arrow.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const Arrow = ({ direction, onClick, height, width }) => {
```

Comment:
Is it personal preference to write the function as such instead of 

```
export default function Arrow() {
...
}
```
Just wondering

Reply:
Pretty much. Plus, the this keyword, which I don't really use much ironically.